ui/src/app/pages/data/cohort/search-bar.spec.tsx

```diff
@@ -39,6 +39,7 @@ describe('SearchBar', () => {
         setInput={() => {}}
       />
     );
-    expect(wrapper).toBeTruthy();
+    screen.logTestingPlaygroundURL();
+    expect(screen.getByRole('textbox')).toBeTruthy();
```

Comment:
is the SearchBar a textbox, or is there one inside it?  unsure about whether this makes sense for a test

Reply:
The SearchBar has a textbox inside.

The previous test `expect(wrapper).toBeTruthy();` was a way to test that the component rendered.

There does not seem to be an analogous "check that component exists" in RTL.

This component has a lot of variability in it (different parts hiding and showing based on different conditions), but one thing that always should exist (so I was considering as a good stand in for "did this component render) is the TextInput that should always be part of the rendered component:
![image](https://github.com/all-of-us/workbench/assets/24514630/dc0b494f-2392-4ec8-b747-1999d78b2a97)

I suspect that we can certainly do more robust testing here, but I am trying to think of the closest match to what we already have, and then we can build on it later.

An alternate strategy would be to put a data-test-id on the outer div and query based on that, but I am trying to avoid wherever possible unless the existing tests already rely on that convention.

What are your thoughts?

Reply:
Is there an accessible way to go here?  "title" or "name" for this text input?

Reply:
Good call! I updated the test to pass in a search term, so we can select based on that term:
![image](https://github.com/all-of-us/workbench/assets/24514630/5135f6e7-4f69-440e-ac3f-e776d2cab080)